src/engine.rs

```diff
@@ -94,6 +94,22 @@ impl ExitIntoResult for ExitReason {
     }
 }
 
+//TODO: check as I added `Debug` trait there. Perhaps this wasn't added earlier intentionally.
```

Comment:
It wasn't needed previously. What do we need it for now?

Reply:
This is needed for the following piece of code: https://github.com/aurora-is-near/aurora-engine/blob/7ac9995edfa685dc4179f3ba11fe8d84ee2c112e/src/prover.rs#L133
The other approach is to use `sdk_unwrap()`. For that we need to move `SdkUnwrap` trait out of the struct and have it without `contract` feature. Then we need to introduce conditional compilation for its implementations (as the `sdk` module is not available without the `contract` feature.

So I think just adding the `Debug` and use `unwrap` just for that specific case is easier approach and is not that messy.

Reply:
Sure, it's ok to have `Debug` and `unwrap` for now.

I think some day (probably in the medium to long term) we should go through and eliminate all `unwrap` / `expect` calls in wasm code in favour of the sdk versions. This should make the binary smaller, which will be nice. But it won't make a difference right now because the majority of the size comes from secp256 stuff.

Reply:
I agree. For this specific case, I thought that `Debug` was not added intentionally to force everyone to use `sdk_unwrap()` instead.